### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SebastianMeler**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 numpy
 hyperopt
-tensorflow-gpu==1.13.1
+tensorflow==1.13.1
```
Comment (MudlaffP) [inline requirements.txt]: here `tensorflow-gpu` should stay

**Review by mkulaczkowski — CHANGES_REQUESTED** [review]
@SebastianMeler This would again create the issues - As mentioned we should remove the tensorflow from main requirements all together and add that to requirmentes-local file

**Comment by EnisBerk** [comment]
[enisberk/hyperopt-keras-sample](https://github.com/enisberk/hyperopt-keras-sample)
@SebastianMeler @mkulaczkowski I removed tensorflow from main requirements, I placed cpu/gpu requirements under the local_infra, updated make file accordingly. 
I also updated docker files, but I need reviews on those. 

Also docker files do not have something equivalent to that ? :
`hyperopt-mongo-worker --mongo $(MONGO_DB_HOST):$(MONGO_DB_PORT)/$(EXPERIMENT_NAME) --exp-key=$(EXPERIMENT_NAME) --last-job-timeout=30  --reserve-timeout=200`

**Review by MudlaffP — COMMENTED** [review]
@EnisBerk at this moment if we need `tensorflow-gpu` we need docker image with cuda drivers otherwise there will be error during command run.

We do not need two different docker images. We need one docker image to run `hyperopt` and `hyperopt-mongo-worker`. By passing command in `docker-compose.yml` we can run one docker image with different responsibility and do not hold two almost identical docker images in repo.

@SebastianMeler simply change things to work that way. He will commit changes with working local version of hyperopt-keras with gpu and cpu support and only then this PR will be resolved.

As far as I understand it should work that way that if there is gpu then it should use gpu but gpu is not detected it should use cpu to train and test models. And that is our target.

**Review by MudlaffP — APPROVED** [review]

**Comment by EnisBerk** [comment]
@MudlaffP 
Using only tensorflow-gpu and docker image with GPU support makes sense to me. 
I used two docker images because that's how this situation was handled at [mnist-sample](https://github.com/Paperspace/mnist-sample).

**Review by mkulaczkowski — APPROVED** [review]

**Pull request merged**